From 6aa1160d16b0886af125a7e7883aab2203ed156d Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Tue, 4 Aug 2020 23:24:03 -0700
Subject: [PATCH 004/267] Sync regex.h from Gnulib
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

* posix/regex.h: Remove an ‘#ifndef _CRAY’ that hasn’t been needed
for years in Gnulib (and was needed only because of Gnulib).
---
 posix/regex.h | 2 --
 1 file changed, 2 deletions(-)

diff --git a/posix/regex.h b/posix/regex.h
index 87cce7f5cb..5fe41c8685 100644
--- a/posix/regex.h
+++ b/posix/regex.h
@@ -600,11 +600,9 @@ extern void re_set_registers (struct re_pattern_buffer *__buffer,
 #endif	/* Use GNU */
 
 #if defined _REGEX_RE_COMP || (defined _LIBC && defined __USE_MISC)
-# ifndef _CRAY
 /* 4.2 bsd compatibility.  */
 extern char *re_comp (const char *);
 extern int re_exec (const char *);
-# endif
 #endif
 
 /* For plain 'restrict', use glibc's __restrict if defined.
-- 
2.29.0

